docs/commands/extract.md

```diff
@@ -17,11 +17,16 @@ genome / transcriptome without library prep artifacts affecting
 the alignment process.  This effectively extracts the cDNA 
 portion of each read.
 
-The cDNA portion of each read is expected to be in sections that
+If a given model has a `coding_region` defined, `extract` will 
+use any sections labeled with the same name as thhe `coding_region`
+as the section of the read to retain during the extraction process.  
+
+For models that do not have a `coding_region` defined, the 
+cDNA portion of each read is expected to be in sections that
 Longbow has labeled as `random`.  The leading and trailing 
 sequences that `extract` uses as markers are configurable and 
 default to the regions that most commonly flank the cDNA regions
-in the current models (`10x_adapter` and `Poly_A`).
+in the current models (`BOREAS` and `Poly_A`).
```

Comment:
Fine for now, but we should go back and reevaluate what we should be naming these adapters.

Reply:
Yeah - I'm not a huge fan of these new adapter names either.  Sounds good.

Reply:
Added issue #87 to track this.